sdk/cosmosdb/cosmos/src/client/Item/Item.ts

```diff
@@ -272,4 +321,128 @@ export class Item {
       );
     }, this.clientContext);
   }
+
+  async encryptItem<T extends ItemDefinition>(item: T): Promise<T> {
```

Comment:
is this function same as the one defined in Items.ts? If yes, we could move these to common and import.